src/js/pre-need/config/form.js

```diff
@@ -166,7 +167,7 @@ const formConfig = {
                       'maritalStatus',
                       'militaryStatus'
                     ], _.set('militaryStatus.enum', [
-                      'V', 'R', 'A', 'E', 'S', 'O', 'X'
+                      'A', 'R', 'S', 'V', 'X', 'E', 'O'
```

Comment:
<img width="1071" alt="screen shot 2017-11-14 at 11 55 00 pm" src="https://user-images.githubusercontent.com/16051172/32824843-6dd1fddc-c997-11e7-8996-65435a177832.png">

Reply:
@stacyhlinn Is this a place where we should use a dropdown instead? It's pretty long.

Reply:
Good idea @jbalboni, @stacyhlinn here is a screenshot to assist in your deliberation:
<img width="528" alt="screen shot 2017-11-15 at 9 58 23 pm" src="https://user-images.githubusercontent.com/16051172/32876109-2d1784c4-ca50-11e7-8692-242a9d99882b.png">
<img width="531" alt="screen shot 2017-11-15 at 9 57 58 pm" src="https://user-images.githubusercontent.com/16051172/32876112-2f25c672-ca50-11e7-89ba-3795c93d2634.png">

Reply:
@cehsu  per the earlier issue to alphabetize the military status, can we order as the following:
Active Duty
Reserve/National Guard
Retired
Retired Active Duty
Retired Reserve/National Guard
Veteran 
Other

I know it's not truly in alpha order since "Other" is last, but it seems odd to have "Other" in the middle

Reply:
@jbalboni agreed. The amount of selections should warrant a dropdown.

Reply:
Sounds good @peggygannon @stacyhlinn, kindly refer to this updated screenshot:
<img width="534" alt="screen shot 2017-11-16 at 11 08 07 pm" src="https://user-images.githubusercontent.com/16051172/32935051-1edb4d36-cb23-11e7-9a55-430247137bab.png">